ote/src/clj/ote/services/operators.sql

```diff
@@ -44,3 +44,15 @@ SELECT u.id as "user-id"
 SELECT "ckan-group-id" FROM "transport-operator" op
  WHERE op."id" = :id
  LIMIT 1;
+
+-- name: services-associated-to-operator
+-- FETCH ALL SERVICES WHERE GIVEN OPERATOR IS ASSOCIATED.
```

Comment:
No need for shouting

Reply:
Changed it to reasonable conversational volumes.